java/ql/test/library-tests/dataflow/taintsources/IntentSources.java

```diff
@@ -1,6 +1,7 @@
 package com.example.myapp;
 
 import android.app.Activity;
+import android.content.Context;
```

Comment:
Does this add an unused import, or did the class not compile previously?

Reply:
It's indeed unused - lingering code from a change in stubs I later undid. Fixed in 1c7101951ae5dc5b5c80e5d59f0fb36bc313741d, thanks!